### PR TITLE
Migrate build from Maven to Gradle

### PR DESCRIPTION
**Pull request by cbeams**
Per bisq-network/dao#41

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -46,7 +46,7 @@ dependencies {
     compile('com.google.inject:guice:4.1.0') {
         exclude(module: 'guava')
     }
-    compile('com.github.bisq-network.libdohj:libdohj-core:01ae7770') {
+    compile('network.bisq.libdohj:libdohj-core:01ae7770') {
```
Comment (ManfredKarrer) [inline build.gradle]: It requires changes in libdohj as well, no?

**Review by ManfredKarrer — COMMENTED** [review]
When I do a 
'gradle install'

I get that error:
* What went wrong:
A problem occurred evaluating root project 'bisq-common'.
> Could not find method annotationProcessor() for arguments [org.projectlombok:lombok:1.16.16] on object of type org.gradle.api.internal.artifacts.dsl.dependencies.DefaultDependencyHandler.

**Review by ManfredKarrer — APPROVED** [review]
Ah after
'./gradlew build'
it worked. 

ACK

**Review by ManfredKarrer — APPROVED** [review]

File: build.gradle (hunk 1 above)
Comment (cbeams) [reply build.gradle]: Did you mean to ask whether changing the `groupid` like we see above requires changes in the libdohj build?

If so, the answer is no. The `network.bisq.*` groupids we're using here are a feature of JitPack we're taking advantage of. See https://jitpack.io/docs/#custom-domain-name, and see this DNS TXT entry we have set up in line with those docs:

```
$ dig txt git.bisq.network | grep github
git.bisq.network.       259     IN      TXT     "https://github.com/bisq-network"
```

**Pull request merged**